pointerevents/compat/pointerevent_touch-action_two-finger_interaction-manual.html

```diff
@@ -0,0 +1,100 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Pointer Event: touch-action test for two-finger interaction</title>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type"/>
+    <link rel="author" title="Google" href="http://www.google.com "/>
+    <meta name="assert" content="Tests that a two-finger pan gesture is cancelled in 'touch-action: pan-x pan-y' but is allowed in 'touch-action: pinch-zoom'"/>
+    <link rel="stylesheet" type="text/css" href="../pointerevent_styles.css">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script type="text/javascript" src="../pointerevent_support.js"></script>
+    <script type="text/javascript">
+      var detected_pointertypes = {};
+      var event_log = [];
+      var active_pointers = 0;
+
+      function resetTestState() {
+        detected_eventTypes = {};
+        event_log = [];
+        active_pointers = 0;
+      }
+
+      function run() {
+        var test_pointer_event = setup_pointerevent_test("touch-action test for two-finger interaction", ["touch"]);
+
+        on_event(document.getElementById("done"), "click", function() {
+          test_pointer_event.step(function () {
+            assert_equals(active_pointers, 0);
+
+            var expected_events =
+              "pointerdown@black, pointerdown@black, pointercancel@black, pointercancel@black, " +
+              "pointerdown@grey, pointerdown@grey, pointerup@grey, pointerup@grey";
+            assert_equals(event_log.join(", "), expected_events);
+          });
+          test_pointer_event.done();
+        });
+
+        var targets = [document.getElementById("black"), document.getElementById("grey")];
+
+        ["pointerdown", "pointerup", "pointercancel"].forEach(function(eventName) {
+          targets.forEach(function(target){
+            on_event(target, eventName, function (event) {
+              detected_pointertypes[event.pointerType] = true;
+              event_log.push(event.type + "@" + event.target.id);
+
+              if (event.type == "pointerdown") {
+                active_pointers++;
+                event.target.setPointerCapture(event.pointerId);
```

Comment:
I originally wanted to have small divs with capturing so that the swipe can go off the div & still pass the test. This would perhaps be better for testing on small screens.

Current layout of the divs are good for small screen, so removed the capturing.